app/components/file_attachment_list/file_attachment.js

```diff
@@ -108,7 +108,7 @@ export default class FileAttachment extends PureComponent {
                     onLongPress={onLongPress}
                 >
                     <FileAttachmentImage
-                        file={data || {}}
+                        file={data}
```

Comment:
Oh, I missed that. I'd rather not do this because it'll be copied in other code, so I'm going to see if FileAttachmentImage can either user defaultProps or work with an undefined file

Reply:
I'm just going to re-add that for now since this needs to get in for 1.19